lib/ddtrace/contrib/resque/resque_job.rb

```diff
@@ -0,0 +1,30 @@
+require 'ddtrace/ext/app_types'
+
+module Datadog
+  module Contrib
+    # Uses Resque job hooks to create traces
+    module ResqueJob
+      SERVICE = 'resque'.freeze
+
+      def self.extended(base)
+        Datadog::Pin.new(SERVICE, app_type: Ext::AppTypes::WORKER).onto(base)
```

Comment:
we attach the `Pin` to the base class, but we don't provide a clean way to change the service name. Can you write here an example of how I change the service name from the main app? Let's say I have Rails + Resque, how do I change the service name from `resque` to `my-app-at-resque` from a Rails initializer?